app/views/games/index.html.erb

```diff
@@ -1 +1,30 @@
-<p>welcome to our chess app<p>
\ No newline at end of file
+<% @available_games = Game.available %>

```

Comment:
Move this line into the controller. games_controller. The `index` action.

It's best practice to put all database queries into the controller.